support/magicleap/README.md

```diff
@@ -29,3 +29,104 @@ This builds the application `.out/Servo2D/Servo2D.mpk`.
 Alternatively, in Visual Studio code, open the `support/magicleap/Servo2D` directory,
 and use the `Terminal/Run Build Task...` menu option to build the
 Servo2D application.
+
+## Debugging gstreamer
+
+By default, Servo links against release builds of gstreamer, even for debug builds,
+so if you want to use gdb on gstreamer, you've got some work to do...
+
+First off, you'll need to build the gstreamer libraries with debug symbols.
+To do this, edit `support/magicleap/gstreamer/mlsdk.txt.in` and add `'-g', '-O0`,` to
+`c_args` and `cpp_args`:
+
+```
+[properties]
+c_args = [
+  '-g', '-O0',
```

Comment:
You should not have to manually write those args. Just pass `--debug` and `--optimization 0` to meson. By default it should already have `-g` and `-O2` args.

Reply:
Sorry, I mean `--buildtype debug` and `--optimization 0`

Reply:
or even just don't pass --optimization, it seems buildtype debug implies no optimization already.

Reply:
Yes, those would probably be better ways of doing it, not sure it's worth a new PR for though, especially since these are mainly notes for future!me.